core/udemy.py

```diff
@@ -77,6 +79,21 @@ def redeem(self, url: str) -> None:
             if element_text not in self.settings.languages:
                 print(f"Course language not wanted: {element_text}")
                 return
+        if self.settings.categories:
+            # If the wanted categories are specified, get all the categories of the course by
+            # scraping the breadcrumbs on the top
+
```

Comment:
Have a variable with the xpath since its used more than once

Reply:
Do I **need** to use the `XPath`? It is working perfectly fine with the `class` name.

Reply:
Sorry I didn't check if it was xpath or class. I just meant to stick it into a variable. Using the class name is fine